charts/incubator/teamspeak/Chart.yaml

```
@@ -1,6 +1,6 @@
 ---
 apiVersion: v2
-appVersion: 3.13.6
+appVersion: 3.13.7
```

```suggestion
appVersion: 3.14.0
```
Since this is not really a bug fix, the version number should be bumped accordingly